### PR TITLE
Add immucore package

### PR DESCRIPTION
**Pull request by Itxaka**
Signed-off-by: Itxaka <itxaka.garcia@spectrocloud.com>

### CODE REVIEW

**Review by mudler — APPROVED** [review]
We need also the dracut package, no?

**Comment by Itxaka** [comment]
> We need also the dracut package, no?

yep, yep, I added the reviewers way too soon, just wanted to see if it build the immucore package lol

**Review by mudler — APPROVED** [review]
:+1: looking good!

File: packages/system/dracut/immucore/build.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+prelude:
+  - apk add git
+  - |
+    PACKAGE_VERSION=${PACKAGE_VERSION%\+*} && \
+    git clone https://github.com/${GITHUB_ORG}/{{ .Values.name }}.git && cd {{.Values.name}} && git checkout v"${PACKAGE_VERSION}"
+
+steps:
+  - mkdir -p /package/usr/lib/dracut/modules.d
+  - mkdir -p /package/etc/dracut.conf.d/
+  - cp {{.Values.name}}/dracut/28immucore /package/usr/lib/dracut/modules.d/28immucore
```
Comment (mudler) [inline packages/system/dracut/immucore/build.yaml:12]:
```suggestion
  - cp -r {{.Values.name}}/dracut/28immucore /package/usr/lib/dracut/modules.d/28immucore
```

File: packages/system/dracut/immucore/build.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+  - cp {{.Values.name}}/dracut/dracut.conf /package/etc/dracut.conf.d/10-immucore.conf
```
Comment (mudler) [inline packages/system/dracut/immucore/build.yaml:13]:
```suggestion
  - cp -r {{.Values.name}}/dracut/dracut.conf /package/etc/dracut.conf.d/10-immucore.conf
```

Comment (Itxaka) [reply packages/system/dracut/immucore/build.yaml:13]: not in this case, we only want to copy the conf

File: packages/system/dracut/immucore/build.yaml (hunk 1 above)
Comment (Itxaka) [reply packages/system/dracut/immucore/build.yaml]: yes, dropped by mistake when changing the dir :clown_face:

**Pull request merged**